docs/data/toolpad/getting-started/quickstart.md

```diff
@@ -0,0 +1,150 @@
+# Quickstart
+
+Get started building a Toolpad app!
+
+<p class="description">
+This page will guide you into creating your very first Toolpad application. We will create a "hello world" type of application that will ask for a name and print a greeting.
+</p>
+
+## Prerequisites
+
+Make sure you have a running Toolpad instance.
+
+1. Download the docker compose file
+
+   ```sh
+   curl -LO https://raw.githubusercontent.com/mui/mui-toolpad/master/docker/compose/docker-compose.yml
+   ```
+
+1. Start the docker compose services
+
+   ```sh
+   docker-compose -f docker/compose/docker-compose.yml up -d
+   ```
+
+Toolpad will be accessible under `http://localhost:3000/`.
+
+## Navigating Toolpad
+
+![Toolpad app](/static/toolpad/app-ui-chrome.png)
+
+There are 4 most frequently used areas:
+
+- `Instance editor` - this is there you define and manage pages, connection configurations, custom component definitions, etc...
+- `Component's library` - drag component instance from this drawer and drop it in `Application editor` to build UI
+- `Application editor` - this is a place where you build your app and immediately preview how it looks like
+- `Properties editor` - modify properties of a different instance
+
+## Building your first application
+
+1. Open Toolpad by navigating to [http://localhost:3000/](http://localhost:3000/)
+
+   ![Apps overview](/static/toolpad/apps-overview.png)
```

Comment:
This screenshot seems broken. 

Aside, does it make sense to add one more level in the `/public/static/toolpad` folder? I feel like it could quickly grow out of proportions.